test/functional/p2p_leak.py

```diff
@@ -35,10 +35,12 @@ def __init__(self):
         super().__init__()
         self.unexpected_msg = False
         self.ever_connected = False
+        self.got_wtxidrelay = False
+        self.got_sendaddrv2 = False
 
     def bad_message(self, message):
         self.unexpected_msg = True
-        self.log.info("should not have received message: %s" % message.msgtype)
+        print("should not have received message: %s" % message.msgtype)
```

Comment:
Is there a reason to replace `self.log.info` with `print`?

Reply:
Yes - I think it would have caused a crash before because we are inside `P2PInterface` which doesn't have a `log` -  only `BitcoinTestFramework` does.